src/components/rainbow/__spec__.js

```diff
@@ -97,7 +96,7 @@ describe('Rainbow', () => {
       <Rainbow
         data-element='bar'
         data-role='baz'
-        data={ {} }/>
+        data={ fromJS({}) }/>
```

Comment:
however this test fails without first ensure it is immutable 👍

Reply:
This must have been failing silently before. It has to be immutable in the rainbow.

function generateConfig(immutableData, title) {
  const data = immutableData.toJS();